main.py

```diff
@@ -23,24 +23,32 @@ def init_background(self, background_symbol, width, height):
             self.image += background_symbol
       
 current_frame = Frame()
+ 
+def draw(x,y,symbol=None,symbols=None):
+    if symbols == None:
+        i = int((height-y) * (width) + (x-1))
+        current_frame.image = current_frame.image[:i] + symbol + current_frame.image[i + 1:]
+    else:
+        for symbol in symbols:
+            i = int((height-y-symbol[1]) * (width) + (x+symbol[0]-1))
```

Comment:
Indexing is off by 1 on x and y placements

Reply:
corrected indexing